src/options/view.rs

```diff
@@ -64,10 +64,8 @@ impl Mode {
 
             if flag.is_some() && flag.unwrap().matches(&flags::GRID) {
                 let _ = matches.has(&flags::GRID)?;
-                let grid = grid::Options::deduce(matches)?;
                 let row_threshold = RowThreshold::deduce(vars)?;
                 let grid_details = grid_details::Options {
-                    grid,
```

Comment:
Could you explain me the reason behind losing this part of the code, is across now working right away, it is a flag that still need to be handled

Reply:
Oh I must've missed where that flag was used in the old version. I'll fix it up :)